Python/Product/PythonTools/PythonTools/Commands/SendToReplCommand.cs

```diff
@@ -159,10 +159,10 @@ class SendToReplCommand : Command {
 
                     var activeView = CommonPackage.GetActiveTextView(_serviceProvider);
 
-                    IPythonInterpreterFactory factory;
-                    if (activeView != null && (factory = activeView.GetInterpreterFactoryAtCaret(_serviceProvider)) != null) {
+                    InterpreterConfiguration config;
+                    if (activeView != null && (config = activeView.GetInterpreterConfigurationAtCaret(_serviceProvider)) != null) {
```

Comment:
is this better?
 if  (config = activeView?.GetInterpreterConfigurationAtCaret(_serviceProvider)) != null)

Reply:
we can do that, i'll change it